unlock-app/src/__tests__/middlewares/providerMiddleware.test.ts

```diff
@@ -2,6 +2,9 @@ import providerMiddleware from '../../middlewares/providerMiddleware'
 import { SET_PROVIDER } from '../../actions/provider'
 import { setError } from '../../actions/error'
 import { FATAL_MISSING_PROVIDER } from '../../errors'
+import Error from '../../utils/Error'
```

Comment:
this is an odd construction, is there a reason you don't `import { Application } from '../../utils/Error'`?

Reply:
You know, in the heat of the moment I forgot that I exported all of those separately as well. I'll update those.